service/src/main/kotlin/fi/espoo/evaka/varda/VardaDevController.kt

```diff
@@ -17,7 +17,7 @@ class VardaDevController(
         db: Database.Connection,
         @PathVariable vardaChildId: Long
     ): ResponseEntity<Unit> {
-        if (System.getenv("VOLTTI_ENV") != "staging") {
+        if (System.getenv("VOLTTI_ENV") == "prod") {
```

Comment:
I'd rather have it the other way round - check if we are in environments where the delete is allowed (shortlist: dev and staging). This way we never ever accidentally delete data by some misconfiguration

Reply:
Agreed, that's somewhat safer way